helix-term/src/ui/editor.rs

```diff
@@ -43,6 +43,8 @@ pub struct EditorView {
     pub(crate) last_insert: (commands::MappableCommand, Vec<InsertEvent>),
     pub(crate) completion: Option<Completion>,
     spinners: ProgressSpinners,
+    // Tracks if the terminal window is focused by reaction to terminal focus events
```

Comment:
```suggestion
    /// Tracks if the terminal window is focused by reaction to terminal focus events
```

so it becomes a doc comment. We don't generate documentation for the helix crates but docs are still nice since they'll show up in LSP hover